bin/release_docker_image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -xe
+
+BASE_TAG=bitjourney/plantuml-service
+TAG=${BASE_TAG}:1.3.3
+
+docker --version
+docker login
```

Comment:
Don't login in the release engineering script. It should already be logged-in.

Reply:
In addition, can you merge the script to `./gradlew release` ask?

Reply:
>  Don't login in the release engineering script. It should already be logged-in.

Why should not login? 
I think it is difficult to force already logged in.
This release script will fail. If not already logged in. So, I force login by `docker login` command.
Is there any other good way?

Reply:
> In addition, can you merge the script to ./gradlew release ask?

OK, I'll try it!

Reply:
As of docker v18.06.0, `docker login` does no-op if it is already logged-in. So the line is not bothersome.